diff --git a/android/src/main/jni/CMakeLists.txt b/android/src/main/jni/CMakeLists.txt
index 1234567..abcdefg 100644
--- a/android/src/main/jni/CMakeLists.txt
+++ b/android/src/main/jni/CMakeLists.txt
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.13)
 set(CMAKE_VERBOSE_MAKEFILE ON)
+find_library(CPP_SHARED_LIB c++_shared)

 set(LIB_LITERAL safeareacontext)
 set(LIB_TARGET_NAME react_codegen_${LIB_LITERAL})
@@ -35,6 +36,7 @@ if (REACTNATIVE_MERGED_SO)
           fbjni
           jsi
           reactnative
+          ${CPP_SHARED_LIB}
   )
 else()
